modules/digital-twins/digital-twins-instances/main.bicep

```diff
@@ -68,6 +68,7 @@ param enableDefaultTelemetry bool = true
 
 @description('Optional. The name of logs that will be streamed.')
```

Comment:
Missing the added suffix. I believe you correctly used search and replace for anything `@sys.description(...)` but did not yet run it for `@description(...)`.

Please update both the templates & readmes

Reply:
Will merge to branch and implement.